### PR TITLE
Wow,Wow! All warnings are fixed!

### PR DESCRIPTION
**Pull request by Tynkute**

### CODE REVIEW

File: Platform.Data.Triplets.Kernel/Common.h [hunk 1]
```diff
@@ -54,14 +54,27 @@ typedef unsigned_integer link_index; // Short for links' array index, unsigned i
...
 
 #define SUCCESS_RESULT 1
-#define succeeded(x) SUCCESS_RESULT == (x)
+#define succeeded(x) (SUCCESS_RESULT == (x))
 #define ERROR_RESULT 0
-#define failed(x) SUCCESS_RESULT != (x)
+#define failed(x) (SUCCESS_RESULT != (x))
 
-signed_integer Error(char* message);
+#endif
```
Comment (poul250) [inline Platform.Data.Triplets.Kernel/Common.h:61]: What is this endif for?)
Good practise is marking them:
```c
#ifdef DEFINE_1
#ifdef DEFINE_2
/* some code... */
#endif // DEFINE_1
#endif // DEFINE_2
```

File: Platform.Data.Triplets.Kernel/Common.h [hunk 2]
```diff
@@ -54,14 +54,27 @@ typedef unsigned_integer link_index; // Short for links' array index, unsigned i
...
+    #define ERROR_MESSAGE_WITH_CODE(message, errorCode)  fprintf(stderr, "%s Error code: %" PRId64 ".\n\n", message, errorCode);
+#else
+    #define ERROR_MESSAGE_WITH_CODE(message, errorCode)
 #endif
+
+#ifdef DEBUG
+    #define DEBUG_MESSAGE(message) fprintf(stdout, "%s\n", message)
+#else
+    #define DEBUG_MESSAGE(message)
+#endif
```
Comment (poul250) [inline Platform.Data.Triplets.Kernel/Common.h:80]: Here you need to add an empty line to the end of the file

File: Platform.Data.Triplets.Kernel/Common.h (hunk 1 above)
Comment (Tynkute) [reply Platform.Data.Triplets.Kernel/Common.h:61]: Thanks, good advice

File: Platform.Data.Triplets.Kernel/PersistentMemoryManager.c [hunk 3]
```diff
@@ -522,32 +577,35 @@ signed_integer CloseStorageFile()
...
         {
-            DebugInfo("Closing storage file...");
+            DEBUG_MESSAGE("Closing storage file...");
 
             // Перед закрытием файла обновляем его размер файла (это гарантирует его минимальный размер).
             ResizeStorageFile();
 
 #if defined(WINDOWS)
-            if (storageFileHandle == INVALID_HANDLE_VALUE) // т.к. например STDIN_FILENO == 0 - для stdin (под Linux)
+            if (storageFileHandle == INVALID_HANDLE_VALUE){
```
Comment (Konard) [inline Platform.Data.Triplets.Kernel/PersistentMemoryManager.c:586]: Here the { should be on next line.

**Pull request merged**